create_rootfs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#Setup nfs server on Centos7
+
+# sudo yum install nfs-utils
+# sudo echo '$NFSROOT     192.168.0.1/24(rw,sync,no_root_squash,no_subtree_check)' >> /etc/exports
+# sudo exports -ra
+
+# sudo firewall-cmd --permanent --add-port=69/udp
+# sudo firewall-cmd --zone=public --add-service=tftp --permanent
+# sudo firewall-cmd --permanent --add-port=2049/tcp
+# sudo firewall-cmd --zone=public --add-service=nfs --permanent
+# sudo systemctl restart firewalld
+
+#Setup nfs server on Debian-like system
+
+sudo mkdir $NFSROOT
+sudo echo "$NFSROOT    192.168.0.1/24(rw,sync,no_root_squash,no_subtree_check)" >> /etc/exports
+sudo chown root:root $NFSROOT
+sudo chmod 755 $NFSROOT
+
+apt-get install nfs-kernel-server nfs-common debootstrap
```

Comment:
@pkubaj thanks for *BSD view, but as comment above state `Setup nfs server on Debian-like system`. Sorry, but I don't see economical value in providing documentation for *BSD in area of PXE. Of course this doesn't mean it will not be needed in future. For now we are stick with Debian-based systems, which seem to be the most popular ones and which are common in 3mdeb. Any adjustment that you have to make to make it work on *BSD systems are on your own and you are welcome to add instructions to documentation.